Dockerfile

```diff
@@ -0,0 +1,14 @@
+FROM node:latest
+
+# Create the docs website directory
+WORKDIR /website
+
+COPY . .
```

Comment:
Is this needed?

Reply:
Yes. The container has to have data from the containing folder (docs/ folder for instance), therfore the image has a copy step that copying the necessary data.

Reply:
And line 8?

Reply:
oops, fixed now